master/static/srv/check_idle.py

```diff
@@ -75,8 +75,7 @@ def main():
     notebook_server = f"https://127.0.0.1:{port}/proxy/{notebook_id}"
     master_url = api.canonicalize_master_url(os.environ["DET_MASTER"])
     cert = certs.default_load(master_url)
-    utp = authentication.login_with_cache(info.master_url, cert=cert)
-    sess = api.Session(master_url, utp, cert)
+    sess = authentication.login_with_cache(master_url, cert=cert)
```

Comment:
This looks like a bugfix. `info` had not been in scope.

Reply:
Decided to not implement a regression test. This doesn't seem like functionality we care much about.